h2o-docs/src/product/data-science/model_selection.rst

```diff
@@ -285,9 +335,13 @@ Examples
       # {‘Intercept’: 4.785482292681689, ‘CAPSULE’: 0.7207023955198935, ‘AGE’: 0.006873599969264931, ‘DPROS’: 0.07827698214607832, ‘DCAPS’: 0.4377770966619996, ‘PSA’: 0.012450143759298283}, 
       # {‘Intercept’: 4.853286962151182, ‘CAPSULE’: 0.7173933092205801, ‘AGE’: 0.00679089119920351, ‘RACE’: -0.06068692599374028, ‘DPROS’: 0.07928808123744804, ‘DCAPS’: 0.4384709133624667, ‘PSA’: 0.012572275831333262}, 
       # {‘Intercept’: 4.852663604264297, ‘CAPSULE’: 0.7153633277776693, ‘AGE’: 0.006948797960002643, ‘RACE’: -0.05843440305164041, ‘DPROS’: 0.07918100130777159, ‘DCAPS’: 0.43531498557623927, ‘PSA’: 0.012606061059188276, ‘VOL’: -0.0005196059470357373}]
+
+      # Retrieve the list of coefficients for a subset size of 3:
+      coeff3 = maxrglmModel.coef(3)
+      # {'Intercept': 5.349021488372978, 'CAPSULE': 0.757501440465183, 'DCAPS': 0.47979554935185015, 'PSA': 0.012890961277678725}
       
       # Retrieve the list of standardized coefficients:
-      coeff_norm = maxrglmModel.coef_norm()
+      coeff_norm = maxrModel.coef_norm()
```

Comment:
please add accessing a single subset for coef_norm() as well like maxrModel.coef_norm(3)